src/config.h

```diff
@@ -64,7 +64,7 @@ struct Config {
   bool logSkippedPathsForIndex = false;
 
   // Maximum workspace search results.
-  int maxWorkspaceSearchResults = 500;
+  size_t maxWorkspaceSearchResults = 500;
```

Comment:
`int` is generally preferred.

Reply:
I think `size_t` is fine to use in a for loop `for (size_t i = 0; i < a.size(); i++)` because STL containers have unsigned `::size()`. Because the scope of a loop variable is limited. But for config variables unsigned types should be avoided

Reply:
I've changed the config options back to `int`.